service/transaction-processor.js

```diff
@@ -542,10 +542,21 @@ async function callSpecialActions(tx) {
         const projectInfo = await projectService.getProjectInfoByWallet(tx.to_wallet, tx.coop_id)
         const projectWalletActivateTx = await repo.findByWalletOrThrow(tx.to_wallet, tx.coop_id)
         if (projectInfo.investmentCap === projectInfo.totalFundsRaised) {
-            mailServiceGrpcClient.sendProjectFullyFunded(projectWalletActivateTx.hash)
+            await amqp.sendMessage(amqp.QUEUE_MAIL_PROJECT_FULLY_FUNDED, { tx_hash: projectWalletActivateTx.hash })
```

Comment:
no need for the **await** here, **projectFullyFunded** message can be sent asynchronously the same way as the **successfullyInvested** message